apps/jobs/sender.py

```diff
@@ -87,9 +87,13 @@ def publish_submission_message(message):
     # increase counter for submission pushed into queue
     submission_pk = message["submission_pk"]
     num_submissions_in_queue.labels(submission_pk, queue_name).inc()
+    num_processed_submissions.labels(submission_pk, queue_name).inc(0)
```

Comment:
Why are we doing this? Only submissions pending for evaluation are published. Incrementing metrics of processed submissions by `0` doesn't make sense. If we are doing this because of the inconsistency in the metric values currently then this is not the correct approach